src/main/resources/l10n/JabRef_el.properties

```diff
@@ -1136,6 +1136,7 @@ Cleanup\ entries=Καθαρισμός καταχωρήσεων
 Automatically\ assign\ new\ entry\ to\ selected\ groups=Αυτόματη ανάθεση νέας καταχώρησης στις επιλεγμένες ομάδες
 %0\ mode=λειτουργία %0
 Move\ DOIs\ from\ note\ and\ URL\ field\ to\ DOI\ field\ and\ remove\ http\ prefix=Μετακίνηση του DOI (Ψηφιακό Αναγνωριστικό Αντικειμένου) από το πεδίο σημείωσης και διεύθυνσης URL, στο πεδίο DOI και αφαίρεση του προθέματος http
+Move\ URLs\ in\ note\ field\ to\ url\ field=
```

Comment:
Please only edit the en file. All other languages are handled by crowdin

Reply:
This was originally fetched from [stackoverflow](https://stackoverflow.com/questions/28185064/python-infinite-loop-in-regex-to-match-url?fbclid=IwAR2aH5PWTwfzilmnfWYDIHmr4IsLRdWkIzNy5dQVggINVZ3n11cEtWmBsM4), but there were some modifications in order for it to be functional. Shall I comment that link in my code?